### PR TITLE
fix: Better mobile styles for swarm

### PR DESCRIPTION
**Pull request by mrv777**
### Description
- Adjusted swarm table styles for better display on mobile
- Limited temp and power decimals
- Adjusted layout padding on mobile

### CODE REVIEW

**Comment by mrv777** [comment]
Also added
- Total hashrate of all devices in swarm
- An orange color on temps over 65C to make the danger stand out

**Comment by mrv777** [comment]
Added a bit more
- Disable the refresh button while refreshing
- Better error handling to prevent an error showing [object Object]
- Zero out a device that failed to update since the data is no longer valid and makes it stand out more

**Comment by mrv777** [comment]
@WantClue Any chance this will make it in 2.3.1? or are we at feature freeze for now and will be next version?

File: main/http_server/axe-os/src/app/components/swarm/swarm.component.html [hunk 1]
```diff
@@ -40,8 +50,8 @@
                 <td>{{axe.hashRate * 1000000000 | hashSuffix}}</td>
                 <td>{{axe.uptimeSeconds | dateAgo}}</td>
                 <td>{{axe.sharesAccepted | number: '1.0-0'}}</td>
-                <td>{{axe.power | number: '1.2-2'}} <small>W</small> </td>
-                <td>{{axe.temp}}°<small>C</small></td>
+                <td>{{axe.power | number: '1.1-1'}} <small>W</small> </td>
+                <td [ngClass]="{'text-orange-500': axe.temp > 65}">{{axe.temp | number: '1.0-1'}}°<small>C</small></td>
```
Comment (eandersson) [inline main/http_server/axe-os/src/app/components/swarm/swarm.component.html:54]: Nit: 66 and above feels a bit too aggressive for me, maybe 68 instead?
image

**Comment by WantClue** [comment]
> Added a bit more
> 
> * Disable the refresh button while refreshing
> * Better error handling to prevent an error showing [object Object]
> * Zero out a device that failed to update since the data is no longer valid and makes it stand out more

Considering it right now. I think this is needed I will test and if it works as expected it gets implemented in 2.3.1

**Pull request merged**